README.md

```diff
@@ -6,21 +6,72 @@
 </p>
 
 <p align="center">
-  End-to-end testing framework for web3 developers building on EVM-compatible chains.
+  GuardianTest is an open-source, e2e testing framework for web3 developers building on EVM-compatible chains.
```

Comment:
"GuardianTest is an open-source, end-to-end testing framework for web3 developers building on EVM-compatible chains"